libraries/push/test/src/main/kotlin/io/element/android/libraries/push/test/notifications/FakeNotificationDrawerManager.kt

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright (c) 2023 New Vector Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.element.android.libraries.push.test.notifications
+
+import io.element.android.libraries.matrix.api.core.RoomId
+import io.element.android.libraries.matrix.api.core.SessionId
+import io.element.android.libraries.push.api.notifications.NotificationDrawerManager
+
+class FakeNotificationDrawerManager : NotificationDrawerManager {
+    private val clearMemberShipNotificationForSessionCallsCount = mutableMapOf<String, Int>()
+    private val clearMemberShipNotificationForRoomCallsCount = mutableMapOf<String, Int>()
```

Comment:
Those 2 maps could be of Type `<SessionId, Int>` instead of `<String, Int>`

Reply:
Not really, the `key` used to access them is a concatenation of `sessionId + roomId` (see `getMembershipNotificationKey`).